src/autocomplete.js

```diff
@@ -89,7 +90,13 @@ class Autocomplete {
 
         this.$firstOpenTimer = lang.delayedCall(function() {
             var initialPosition = this.completionProvider && this.completionProvider.initialPosition;
-            if (this.autoShown || (this.popup && this.popup.isOpen) || !initialPosition) return;
+
+            if ((this.popup && this.popup.isOpen) || !initialPosition) return;
+
+            if (this.autoShown) {
+                this.editor.renderer.$gutterLayer.showSpinner(this.base.row);
```

Comment:
I think `$gutterLayer` is not guaranteed to exist, since you can hide the gutter, right? let's check for it's existence in this case. or maybe even move this logic to `editor` or `renderer` itself to encapsulate the implementation details of the gutter, and from here call `editor.showGutterSpinner` or `editor.renderer.showGutterSpinner`?

Reply:
Moved it to `virtual_renderer`